app/i18n/en/namespaces/global.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+const global = {
+  openwebslides: 'Open Webslides',
+  home: {
+    recentActivity: 'Recent activity',
+  },
+  library: 'Library',
+  createNewTopic: 'Create new topic',
+  profile: 'Profile',
+  users: {
+    profile: 'Preferences',
+  },
+  account: 'Account',
+  topics: 'Topics',
+};
+
+export default global;
```

Comment:
See earlier conversation:
- Navbar subkey
- Things that aren't in the navbar should be moved to a more appropriate namespace.